queue_services/entity-filer/src/entity_filer/filing_processors/continuation_in.py

```diff
@@ -0,0 +1,136 @@
+# Copyright © 2024 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""File processing rules and actions for the continuation in of a business."""
+import copy
+from typing import Dict
+
+from entity_queue_common.service_utils import QueueException
+from legal_api.models import Business, Document, DocumentType, Filing, Jurisdiction
+from legal_api.utils.legislation_datetime import LegislationDatetime
+
+from entity_filer.filing_meta import FilingMeta
+from entity_filer.filing_processors.filing_components import aliases, business_info, filings, shares
+from entity_filer.filing_processors.filing_components.offices import update_offices
+from entity_filer.filing_processors.filing_components.parties import update_parties
+
+
+def create_foreign_jurisdiction(continuation_in: Dict, business: Business, filing: Filing):
+    """Create jurisdiction and director affidavit document."""
+    foreign_jurisdiction = continuation_in.get('foreignJurisdiction')
+
+    jurisdiction = Jurisdiction()
+    jurisdiction.country = foreign_jurisdiction.get('country')
+    jurisdiction.region = foreign_jurisdiction.get('region')
+    jurisdiction.legal_name = foreign_jurisdiction.get('legalName')
+    jurisdiction.identifier = foreign_jurisdiction.get('identifier')
+    incorporation_date = foreign_jurisdiction.get('incorporationDate')
+    jurisdiction.incorporation_date = LegislationDatetime.as_utc_timezone_from_legislation_date_str(incorporation_date)
+    jurisdiction.tax_id = foreign_jurisdiction.get('taxId')
```

Comment:
Some of the above may not exist in the JSON. Is that OK?

Reply:
Yes. get func will return None if not exist and its an optional field in db.